### PR TITLE
Mobile first section

### PR DESCRIPTION
**Pull request by AnelisaDee**
**This is a portfolio mobile version.**
This project is developed using best practice of linters. I created the sections of this mobile website using the template found in Figma App.

**Project Requirements**

**The toolbar (or header)**

- I used flexbox container for the toolbar in order for it to be more responsive in any device.
- Styled it according to the template I chose.
- The headline section (right after the header)
**The content of the body is divided into 2 sections (section and footer)**
- Both sections are inside the flexbox container.
- Added 2 space indentation inside the .css file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+> Build the first 2 sections of the mobile website using the template from Figma. Flexbox is used for the elements on the page.
+
+
+## Built With
+
+-  HTML and CSS
+
+
+
```
Comment (cynthiainga) [inline README.md]: - [ ] Good implementation of the `README`  file 👍 . I suggest you add the `Getting started ` part then in that part give to user prerequisites(things that they need in their device) and also explain to the users how to get your project locally and how they can open it locally. This helps users who don't know how to use github 🙃 🙃

File: README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (cynthiainga) [inline README.md:52]: - [ ] Please kindly add the `MIT LICENSE` file to your directory, then when user clicks on this link they can direct to your `MIT` license and your README file will look professional 😎😎

File: index.html [hunk 3]
```diff
@@ -4,9 +4,47 @@
...
+    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
     <title>Portfolio</title>
 </head>
 <body>
-    
+    <div class="main-container">
+    <header>
+        <nav class="navbar">
+            <h2 class="logo">Anelisa Didishe</h2>
```
Comment (cynthiainga) [inline index.html]: - [ ] I suggest you put your logo into the `a` tag, this is for semantics reasons [check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info 🙃 😉

File: index.html [hunk 4]
```diff
@@ -4,9 +4,47 @@
...
+            <div id="menu">
+                <i class="fa fa-bars"></i>
+            </div>
+        </nav>
```
Comment (cynthiainga) [inline index.html]: - [ ] I suggest you make some changes to your header, your design should stick as much as possible to the Figma design. You should add the padding-top, and try to reduce the size of your hamburger menu and logo 😁 😁

Your design

Screen Shot 2022-10-27 at 17 46 27

Figma design

Screen Shot 2022-10-27 at 17 46 59

File: index.html [hunk 5]
```diff
@@ -4,9 +4,47 @@
...
+            </li>
+            <li class="social">
+              <a href="#"><i class="fa fa-twitter"></i></a>
+            </li>
+            <li class="social">
+              <a href="#"><i class="fa fa-medium"></i></a>
+            </li>
+          </ul>
+        </div>
+    </div>
```
Comment (cynthiainga) [inline index.html]: - [ ] The container of the section is a `section` tag. It represents a generic standalone section of the document, please kindly use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info 😁 😁

File: index.html [hunk 6]
```diff
@@ -4,9 +4,47 @@
...
+            <div id="menu">
+                <i class="fa fa-bars"></i>
+            </div>
+        </nav>
+    </header>
+    <div class="container">
+        <h1>Hey There. I’m Anelisa</h1>
+        <h2>I’am a Software Developer</h2>
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
```
Comment (cynthiainga) [inline index.html:28]: - [ ] Your design should stick as much as possible (e.g. font, colors, images, text, margins) to the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). Please kindly make sure you have the same fonts (font-size, font-family,...) as the Figma design 😁 🙃

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @AnelisaDee, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Good commit history ✔️ 
- [ ] Correct Github flow ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @AnelisaDee, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**